config/v1/types_infrastructure.go

```diff
@@ -182,6 +182,22 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// Interface on a Baremetal server where the provisioning network is connected.
+	ProvisioningInterface string `json:"provisioningInterface"`
+
+	// IP address of the Provisioning interface.
```

Comment:
This isn 't the IP of the interface. It's a VIP that moves with the provisioning service.

```suggestion
	// The VIP to be used by the provisioning service. This must be on the provisioning subnet, and outside of the DHCP range.
```

Reply:
That's not strictly true, it's an IP which we use to configure the IP on whatever node runs the provisioning service, to me VIP implies loadbalancing/keepalived which is not the case here?

Reply:
That's my misunderstanding, then. I thought that it was the IP that moved with the service if it was rescheduled. Maybe that's not the same thing as a VIP.